TYPO3.Flow/Classes/TYPO3/Flow/Core/Booting/Scripts.php

```diff
@@ -161,10 +178,16 @@ public static function initializeSignalSlot(Bootstrap $bootstrap)
      */
     public static function initializePackageManagement(Bootstrap $bootstrap)
     {
-        $packageManager = new \TYPO3\Flow\Package\PackageManager();
-        $bootstrap->setEarlyInstance(\TYPO3\Flow\Package\PackageManagerInterface::class, $packageManager);
+        $packageManager = new PackageManager();
+        $bootstrap->setEarlyInstance(PackageManagerInterface::class, $packageManager);
+
+        // The package:rescan must happen as early as possible, compiletime alone is not enough.
```

Comment:
Something similar is already in the forced cache flushing. Maybe makes sense to wrap the $_SERVER['argv'] somewhere inside the Bootstrap?

Also, is the rescanPackages() method now called twice? Once here in bootstrap and once in the commandController?